src/http_seed_connection.cpp

```diff
@@ -315,7 +315,7 @@ namespace libtorrent
 					}
 
 					// add the redirected url and remove the current one
-					t->add_web_seed(location, web_seed_entry::http_seed);
+					t->add_web_seed(location, web_seed_entry::http_seed, std::string(), web_seed_entry::headers_t(), true);
```

Comment:
iirc, it just means it won't be saved in the resume data, but you still remove the original web seed URL below, won't that be a problem?